docs/site/migration/express-middleware.md

```diff
@@ -6,8 +6,21 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/migration-express-middleware.html
 ---
 
-{% include note.html content="
-This is a placeholder page, the task of adding content is tracked by the
-following GitHub issue:
-[loopback-next#3947](https://github.com/strongloop/loopback-next/issues/3947)
-" %}
+Migrating Express middleware from LoopBack 3 (LB3) application to LoopBack 4
```

Comment:
There seems to be some duplicate content in this file.  Maybe we don't need the first 2 paragraphs?

I'm thinking if it's even better to only have the following content in this file: 
```
There are 2 steps involved in migrating a LoopBack 3 application with Express middleware:
1. ....
2. ....

See [this tutorial(..) for details. 
```

Or can we get the content of the https://github.com/strongloop/loopback-next/blob/master/examples/lb3-application/README.md here?

Reply:
Updated the links.